src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVReferenceUtils.java

```diff
@@ -39,16 +28,18 @@
      * bases in chunk n+1) so that we don't miss any kmers due to the chunking -- we can just kmerize each record
      * independently.
      */
-    public static JavaRDD<byte[]> getRefRDD(final JavaSparkContext ctx,
-                                            final int kSize,
-                                            final ReferenceMultiSource ref,
-                                            final SAMSequenceDictionary readsDict,
-                                            final int ref_record_len,
-                                            final int ref_records_per_partition) {
-        final SAMSequenceDictionary dict = ref.getReferenceSequenceDictionary(readsDict);
-        if ( dict == null ) throw new GATKException("No reference dictionary available");
-
-        final int effectiveRecLen = ref_record_len - kSize + 1;
+    public static JavaRDD<byte[]> getReferenceBasesRDD(final JavaSparkContext ctx,
```

Comment:
I still think this is not general enough to be a utility.... where do you think is it going to use? perhaps it could be made a bit more general if instead of returning chunks of arbitrary length it returns well defined kmer so you would hide the intermediary chanck size.

Reply:
@magicDGS thinks he would have a use for that. 
I think this could be useful as well. 
I'll what improvements (test, representation) I can bring to this.

Reply:
What most using code would find useful is ```JavaRDD<byte[]>``` of all the khmers of a given length rather than a collection of chunks of bases or different sizes.... I guess the reason for returning the later is because to create a JavaRDD it seems that you need to explicitly create a List of all the its elements (is not possible to provide a stream instead?) (so ```ctx.parallelize(kmerList)```... this obviously not practical for a large genome like the human genome. 

I think the best way to approach this by changing the actual RDD that is returned. 
Instead of returning the RDD of the DNA chunks as we do now we return a ```flatMap``` on it that would transform those chunks into the actual kmers so these are only instantiated when they are needed... then the parameters of this method except for the reference source and the kmerSize can be hidden given them reasonable defaults.

Perhaps instead of byte[] you could return SVKmers objects.

I leave it up to you but the chuck-of-DNA RDD return looks a bit nasty and complex as a return type and given its specific about k-1 overlapping ... it is obvious what the intention for it so why don't simply return a RDD to the kmers that is what people really want.

Reply:
I've added a todo for this method, but don't think I have time for it this week or next.
Also cleaned up the documentation a bit.